tests/test_user_security.py

```diff
@@ -0,0 +1,200 @@
+import enum

```

Comment:
cerate a small django app 
user the user security 
make request using postman.  make sure it's working there